lua/nvim-tree/actions/fs/remove-file.lua

```diff
@@ -18,11 +23,13 @@ local function clear_buffer(absolute_path)
   local bufs = vim.fn.getbufinfo { bufloaded = 1, buflisted = 1 }
   for _, buf in pairs(bufs) do
     if buf.name == absolute_path then
-      if buf.hidden == 0 and #bufs > 1 then
+      if buf.hidden == 0 then
```

Comment:
What's the reason for this change?

It is a change in behaviour for non-floating:  the window will be closed even if there are other buffers present.

Reply:
I am not able to reproduce such behaviour. With non-floating window, it seems to behave the same regardless of this change. How can I reproduce this?

Reply:
> I am not able to reproduce such behaviour. With non-floating window, it seems to behave the same regardless of this change. How can I reproduce this?

I should have noted how to reproduce this as I now cannot... the next buffer is correctly displayed.

The window switch and `:bn` are now invoked when there are no other buffers in the window, however that seems fine.

**Real question:** do we need this change? If it's necessary for floating windows it should be done only for the case of floating windows.

Reply:
I reverted this change so non-floating windows should not be affected.

When I was trying to fix the original bug from the referenced issue, the deleted buffer was still displayed for some reason. The important bit here seems to be switching focus to window with soon-to-be-deleted buffer.

Reply:
> When I was trying to fix the original bug from the referenced issue, the deleted buffer was still displayed for some reason. The important bit here seems to be switching focus to window with soon-to-be-deleted buffer.

:+1: